Templates/Kotlin/Operation.stencil

```diff
@@ -1,5 +1,5 @@
 {% if asFile %}
-package {% if moduleName %}{{ moduleName }}{% else %}com.shopify.syrup{% endif %}.{% if isQuery %}queries{% else %}mutations{% endif %}
+package {% if moduleName %}{{ moduleName }}{% else %}com.shopify.syrup{% endif %}.{% if isQuery %}queries{% else %}{{ operation.typeName|lowercase }}s{% endif %}
```

Comment:
I think we should remove `isQuery` and provide the actual types we want without the plural concatenation in the template.

Reply:
@lwise Investigated further as discussed over Slack. 😅 I'm doubling down that we should 🔥 `isQuery`.

You want to reference `operation.type`, not `operation.typeName`. If we aren't happy, or cannot format the string how we want in Stencil, lets format it in Swift, then pass it to stencil to use directly. ie. using plurals for package names like `mutations`, `queries`. To be honest, I can't think of many plurals in package names, so we could probably just change it.